services/utils/formatDate.js

```diff
@@ -0,0 +1,14 @@
+export function formatDate(date, locale = "en-US") {
+  let options = {
+    weekday: "long",
+    day: "numeric",
+    month: "long",
+    year: "numeric",
+    hour: "2-digit",
+    minute: "2-digit",
+    second: "2-digit",
+    timeZoneName: "short",
+  };
```

Comment:
This is not the options used previous, why the change?

Reply:
![6cf7ec37-41ec-4784-a981-20c13dcdd613](https://github.com/EddieHubCommunity/LinkFree/assets/53470896/ecd78886-a1fe-46ee-919c-9966a576babf)

I have added `hour`,`minutes` and `seconds` configs because I though that it would me more precise and also it will give uniform time.

The time would be represented like this -
```
a time of 9:5:3 would be output as 09:05:03:

Hour: 9 becomes 09
Minute: 5 becomes 05
Second: 3 becomes 03
```

Reply:
If this is fine, or should I change it?

Reply:
According to the official docs that type of time is already achieved with the original config 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#examples

Reply:
okay then I will remove `seconds` thing then